Makes ./configure identify texinfo with minor version above 9.

 Jonas Karlsson <jonka750@student.liu.se>

--- binutils-2.18/configure	2007-08-06 20:29:40.000000000 +0000
+++ binutils-2.18/configure.new	2008-02-07 12:12:56.000000000 +0000
@@ -6128,7 +6128,7 @@
     # For an installed makeinfo, we require it to be from texinfo 4.4 or
     # higher, else we use the "missing" dummy.
     if ${MAKEINFO} --version \
-       | egrep 'texinfo[^0-9]*([1-3][0-9]|4\.[4-9]|[5-9])' >/dev/null 2>&1; then
+       | egrep 'texinfo[^0-9]*([1-3][0-9]|4\.([4-9]|[1-9][0-9]+)|[5-9])' >/dev/null 2>&1; then
       :
     else
       MAKEINFO="$MISSING makeinfo"
